static/js/tabs.js

```diff
@@ -1,52 +1,43 @@
+    };
 
-    if(newspapersheet_form > 0 || twittersheet_form > 0 || radiosheet_form > 0 || televisionsheet_form > 0 || internetnewssheet_form > 0){
-        // Rename the save and continue button to Next
-        let cont = $('input[name=_continue]').val('Next Tab')
-        $(document).ready(function() {
-            module = $('.changeform-tabs li.selected').index();
-            let last_tab = $('.changeform-tabs li:last-child').hasClass('selected');
-            if(last_tab) {
-                $('input[name=_addanother]').show()
-                $('input[name=_save]').show()
-                $('input[name=_continue]').hide()
-            }else{
-                $('input[name=_addanother]').hide()
-                $('input[name=_save]').hide()
-                $('input[name=_continue]').show()
-            }
-        });
-        $('body').click(function(e) {
-            setTimeout(function() {
-                module = $('.changeform-tabs li.selected').index();
-                let last_tab = $('.changeform-tabs li:last-child').hasClass('selected');
-                if(last_tab) {
-                $('input[name=_addanother]').show()
-                $('input[name=_save]').show()
-                $('input[name=_continue]').hide()
-                }else{
-                    $('input[name=_addanother]').hide()
-                    $('input[name=_save]').hide()
-                    $('input[name=_continue]').show()
+    $(document).ready(function() {
+        var newspapersheet_form = $('#newspapersheet_form').length > 0;
+        var radiosheet_form = $('#radiosheet_form').length > 0;
+        var televisionsheet_form = $('#televisionsheet_form').length > 0;
+        var internetnewssheet_form = $('#internetnewssheet_form').length > 0;
+        var twittersheet_form = $('#twittersheet_form').length > 0;
+        
+        if(newspapersheet_form || twittersheet_form || radiosheet_form || televisionsheet_form || internetnewssheet_form){
+            // Rename the save and continue button to Next
+            var continueButton$ = $('input[name=_continue]');
+            continueButton$.val('Next Tab')
+            continueButton$.click((function(e) {
+                e.preventDefault();
+                // Move to next tab unless we are on the last tab
+                if ($('.changeform-tabs li.selected:last-child').length === 0){
+                    $('.changeform-tabs li.selected').removeClass('selected').next().addClass('selected');
+                    set_button_navigation();
                 }
-            }, 0)            
-        });
-        $('input[name=_continue]').click((function(e) {
-            e.preventDefault();
-            // Move to next tab unless we are on the last tab
-            if ($('.changeform-tabs li.selected:last-child').length == 0){
-                $('.changeform-tabs li.selected').removeClass('selected').next().addClass('selected');
-                $(`.module_${module}`).removeClass('selected');
```

Comment:
Removing this prevents the `Next Tab` from changing the displayed form to what is currently showing as selected tab. 
![Screenshot 2020-09-18 at 08 02 44](https://user-images.githubusercontent.com/13068580/93558200-8c6ad680-f985-11ea-827a-8dce4e4fd04d.png)

Reply:
Can you add a bit more info @esirK ? What is "module"? Is it our doing is is Jet?

Reply:
yes @kilemensi  it is a Jet class.
They're 2 moving parts involved to have the desired result. 1. The `tabs` themselves and the `modules`
we, therefore, need to move both of them.
![Screenshot 2020-09-18 at 09 05 24](https://user-images.githubusercontent.com/13068580/93562240-51b96c00-f98e-11ea-87f8-52ee28f1e126.png)

Reply:
:+1:  @esirK 

-----

But unless we're sure browsers like Microsoft Edge, etc., out there supports certain JS features, I'd say lets stick to using the old and boring code e.g. string templates vs string concat. Otherwise, a poor coordinator out there will get stuck due to some feature not being supported by their browser.